.github/workflows/wf-device-camera.yml

```diff
@@ -43,10 +46,14 @@ jobs:
     runs-on: ubuntu-latest
     needs:  build-snap
     steps:
-      - name: Kick off Launchpad build
-        uses: canonical/edgex-launchpad-build-action@v1.4
-        with:
-          edgex_snap: "edgex-device-camera"
-          consumer_name: ${{ secrets.LP_CONSUMER_NAME }}
-          access_token: ${{ secrets.LP_ACCESS_TOKEN }}
-          access_secret: ${{ secrets.LP_ACCESS_SECRET }}
+    - name: Kick off Launchpad build
```

Comment:
Looks like the formatting was updated too, was this intentional?

Reply:
It was - It wasn't uniform across the three steps so I cleaned that up. Perhaps that should have been left out of this commit though as it's not related to the change which is actually being made.